server/src/main/java/org/opensearch/action/support/master/info/TransportClusterInfoAction.java

```diff
@@ -59,4 +61,16 @@ public TransportClusterInfoAction(
         super(actionName, transportService, clusterService, threadPool, actionFilters, request, indexNameExpressionResolver);
     }
 
+    @Override
```

Comment:
`@Deprecated` ?

Reply:
Thank you! This is a good catch! I believe you are correct.

Reply:
Changed in the new commit https://github.com/opensearch-project/OpenSearch/pull/3617/commits/a77d350f727579f6f7a9acd5bcd3e474d8cbc615